spec/requests/email_domain/delete_spec.rb

```diff
@@ -0,0 +1,37 @@
+describe "DELETE /authorised_email_domains/:id", type: :request do
+  let(:admin_user) { create(:user, super_admin: true) }
+  let(:normal_user) { create(:user, super_admin: false) }
+  let!(:email_domain_1) { create(:authorised_email_domain, name: 'some.domain.org.uk') }
+  let!(:email_domain_2) { create(:authorised_email_domain, name: 'police.uk') }
+
+  context "when the user is a super admin" do
+    before do
+      https!
+      login_as(admin_user, scope: :user)
+    end
+
+    it "deletes the email domain" do
+      expect {
+        delete admin_authorised_email_domain_path(email_domain_1)
+      }.to change { AuthorisedEmailDomain.count }.by(-1)
+    end
+
+    it 'publishes the authorised domains to S3' do
+      expect_any_instance_of(Gateways::S3).to receive(:upload)
+      delete admin_authorised_email_domain_path(email_domain_1)
+    end
+  end
+
+  context "when the user is not super admin" do
+    before do
+      https!
+      login_as(normal_user, scope: :user)
```

Comment:
Rather than having `let(:normal_user) { create(:user, super_admin: false) }` right up the top of the file and then referencing it here, you could make this a bit more readable by creating the normal user right here where you need it.

Eg:
```
before do
   https!
   login_as( create(:user), scope: :user)
end
```

Also you should use the method `sign_in_user user` not `login_as(user)`. Its defined in `features/support.sign_up_helpers.rb`.